io.github.dosbox-staging.yml

```diff
@@ -0,0 +1,52 @@
+app-id: io.github.dosbox-staging
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+
+command: dosbox
+
+build_options:
+  cppflags: -DNDEBUG
+  cflags: -O3
+  cxxflags: -O3
+
+finish-args:
+  - --device=all # we need OpenGL and controller access
+  - --share=network # IPX and serial port emulation over UDP
+  - --socket=x11 # default to X11, as Gnome does not provide SSD
+  - --share=ipc # necessary for x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --filesystem=home
+
+modules:
+
+  # - "shared-modules/linux-audio/fluidsynth2.json"
+
+  - name: dosbox-staging
+
+    sources:
+      - type: archive
+        url: https://github.com/dosbox-staging/dosbox-staging/archive/v0.76.0.tar.gz
+        sha256: 7df53c22f7ce78c70afb60b26b06742b90193b56c510219979bf12e0bb2dc6c7
+
+    config-opts:
+      - --disable-fluidsynth
+
+    make-args:
+      - V=1
+
+    post-install:
+      # icons
+      - make -C contrib/icons hicolor
+      - install -p -D -m 0644 "contrib/icons/hicolor/16x16/apps/dosbox-staging.png"    "/app/share/icons/hicolor/16x16/apps/${FLATPAK_ID}.png"
+      - install -p -D -m 0644 "contrib/icons/hicolor/22x22/apps/dosbox-staging.png"    "/app/share/icons/hicolor/22x22/apps/${FLATPAK_ID}.png"
+      - install -p -D -m 0644 "contrib/icons/hicolor/24x24/apps/dosbox-staging.png"    "/app/share/icons/hicolor/24x24/apps/${FLATPAK_ID}.png"
+      - install -p -D -m 0644 "contrib/icons/hicolor/32x32/apps/dosbox-staging.png"    "/app/share/icons/hicolor/32x32/apps/${FLATPAK_ID}.png"
+      - install -p -D -m 0644 "contrib/icons/hicolor/scalable/apps/dosbox-staging.svg" "/app/share/icons/hicolor/scalable/apps/${FLATPAK_ID}.svg"
```

Comment:
Just SVG will do, flatpak knows what to do with it.

Reply:
Small icons have a slightly adjusted design to look sharper in small sizes, they are not the same icon.